File: POO/J/OzCorp/src/br/com/ozcorp/business/Engineer.java
package br.com.ozcorp.business;
public class Engineer extends Employee {
	public Engineer(String name, String rg, long cpf, String enrollment, String email, String password,
			BloodType blood, Gender gender) {
		super(name, rg, cpf, enrollment, email, password, Role.Engineer, blood, gender);
	}
}

File: POO/CS/OzCorp/OzCorp/Business/Engineer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OzCorp.Business
{
    public class Engineer : Employee
    {
        public Engineer(string Name, string RG, long CPF, string Enrollment, string Email, string Password, BloodType Blood, Gender Gender) : base(Name, RG, CPF, Enrollment, Email, Password, Role.GetRole(Roles.Engineer), Blood, Gender)
        {
        }
    }
}
